pkg/pod/status.go

```diff
@@ -281,7 +281,8 @@ func getFailureMessage(pod *corev1.Pod) string {
 	for _, status := range pod.Status.ContainerStatuses {
 		term := status.State.Terminated
 		if term != nil && term.ExitCode != 0 {
-			return fmt.Sprintf("%q exited with code %d (image: %q); for logs run: kubectl -n %s logs %s -c %s",
+			return fmt.Sprintf(`%q exited with code %d (image: %q); for logs run:
```

Comment:
Given that we are formatting these messages specifically for user-consumption I think we should add a comment to that effect here.  Otherwise it will be very easy for a dev to mistakenly "correct" this formatting in a future PR without realising why we did it this way in the first place.

Reply:
Will update this with the necessary commit changes and the comment so that these changes are not "corrected".